src/components/forms/CloudDatabaseForm.js

```diff
@@ -0,0 +1,273 @@
+            label="Memory (GB)"
+            id={this.props.data.name + "-db-memory"}
+            value={this.state.memory}
+            allowEmpty
+            placeholder={this.props.memoryMin}
+            min={this.props.memoryMin}
+            max={this.props.memoryMax}
+            onChange={this.handleInputChange}
+            name="memory"
+            hideSteppers
+            invalid={isRangeInvalid(
+              this.state.memory,
+              this.props.memoryMin,
+              this.props.memoryMax,
+            )}
+            invalidText={`RAM must be a minimum of ${this.props.memoryMin}GB and a maximum ${this.props.memoryMax}GB per member`}
+            className="fieldWidthSmaller leftTextAlign"
+          />
+          {/* disk number input */}
+          <NumberInput
+            label="Disk (GB)"
+            id={this.props.data.name + "-db-disk"}
+            value={this.state.disk}
+            allowEmpty
+            placeholder={this.props.diskMin}
+            min={this.props.diskMin}
+            max={this.props.diskMax}
+            onChange={this.handleInputChange}
+            name="disk"
+            hideSteppers
+            invalid={isRangeInvalid(
+              this.state.disk,
+              this.props.diskMin,
+              this.props.diskMax,
+            )}
+            invalidText={`Disk must be a minimum of ${this.props.diskMin}GB and a maximum ${this.props.diskMax}GB per member`}
+            className="fieldWidthSmaller leftTextAlign"
+          />
+          {/* cpu text input */}
+          <NumberInput
+            label="CPU"
+            id={this.props.data.name + "-db-cpu"}
+            value={this.state.cpu}
+            allowEmpty
+            placeholder={this.props.cpuMin}
+            min={this.props.cpuMin}
+            max={this.props.cpuMax}
+            onChange={this.handleInputChange}
+            name="cpu"
+            hideSteppers
+            invalid={this.props.invalidCpuCallback(this.state, this.props)}
+            invalidText={this.props.invalidCpuTextCallback(this.state, this.props)}
+            className="fieldWidthSmaller leftTextAlign"
+          />
+        </IcseFormGroup>
+        <IcseFormGroup>
+          {/* Select Encryption Key */}
+          <IcseSelect
+            value={this.state.encryption_key}
+            groups={this.props.encryptionKeys}
+            formName={this.props.data.name + " CloudDatabase"}
+            name="encryption_key"
+            labelText="(Optional) Encryption Key"
+            handleInputChange={this.handleInputChange}
+            disableInvalid
+            className="fieldWidthSmaller"
+          />
+        </IcseFormGroup>
+      </div>
+    );
+  }
+}
+
+CloudDatabaseForm.defaultProps = {
+  data: {
+    name: "",
+    resource_group: "",
+    use_data: false,
+    plan: "standard",
+    encryption_key: "",
+    service: "",
+    group_id: "member",
+    memory: "",
+    disk: "",
+    cpu: "",
+    memoryMin: "",
```

Comment:
I don't think we need memoryMin/Max, diskMin/Max, or cpuMin/Max in the data object. can be removed

Reply:
I moved these out of the data object, but forgot to delete them from the data object, doing now!